engines/optix/OptiXScene.cpp

```diff
@@ -40,16 +40,22 @@ OptiXScene::OptiXScene(AnimationParameters& animationParameters,
     , _lightBuffer(nullptr)
 {
     _backgroundMaterial = std::make_shared<OptiXMaterial>();
+    auto oc = OptiXContext::get().getOptixContext();
 
     { // Create dummy texture sampler
-        auto oc = OptiXContext::get().getOptixContext();
         ::optix::TextureSampler sampler = oc->createTextureSampler();
         sampler->setArraySize(1u);
         optix::Buffer buffer =
             oc->createBuffer(RT_BUFFER_INPUT, RT_FORMAT_FLOAT4, 1, 1);
         sampler->setBuffer(buffer);
         _dummyTextureSampler = sampler;
     }
+
+    { // Create dummy simulation data
```

Comment:
Add why you need this workaround, otherwise somebody may try to waste time trying to remove it.